components/camel-debezium/src/main/docs/debezium-component.adoc

```diff
@@ -0,0 +1,217 @@
+[[debezium-component]]
+= Debezium Component
+
+*Available as of Camel version 3.0*
+
+The Debezium component is wrapper around https://debezium.io/[Debezium] using https://debezium.io/docs/embedded/[Debezium Embedded], which enabled Change Data Capture from various databases without the need of Kafka or Kafka Connect.
+
+*Note on handling failures:* Per https://debezium.io/docs/embedded/#handling_failures[Debezium Embedded Engine] documentation, the engines is actively recording source offsets and periodically flushes these offsets to a persistent storage, so when the application is restarted or crashed, the engine will resume from the last recorded offset.
+Thus, at normal operation, your downstream routes will receive exactly once event, however in case of application crash (not having a graceful shutdown), the application will resume from the last recorded offset,
+which may result of receiving duplicate events immediately after the restart. Therefore, your downstream routes should be tolerant enough of such case and deduplicate events if needed.
+
+Maven users will need to add the following dependency to their `pom.xml`
+for this component.
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-debezium</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+------------------------------------------------------------
+
+== URI format
+
+[source,java]
+---------------------------
+debezium:connector-type[?options]
+
+---------------------------
+
+== Supported Debezium Connectors
+- https://debezium.io/docs/connectors/mysql/[MySql].
+
+
+== Options
+
+
+// component options: START
+The Debezium component supports 2 options, which are listed below.
+
+
+
+[width="100%",cols="2,5,^1,2",options="header"]
+|===
+| Name | Description | Default | Type
+| *configuration* (consumer) | Allow pre-configured Configurations to be set, you will need to extend EmbeddedDebeziumConfiguration in order to create the configuration for the component |  | EmbeddedDebezium Configuration
+| *basicPropertyBinding* (advanced) | Whether the component should use basic property binding (Camel 2.x) or the newer property binding with additional capabilities | false | boolean
+|===
+// component options: END
+
+
+// endpoint options: START
+The Debezium endpoint is configured using URI syntax:
+
+----
+debezium:connectorType
+----
+
+with the following path and query parameters:
+
+=== Path Parameters (1 parameters):
+
+
+[width="100%",cols="2,5,^1,2",options="header"]
+|===
+| Name | Description | Default | Type
+| *connectorType* | *Required* The Debezium connector type that is supported by Camel Debezium component. |  | String
```

Comment:
Should this document the allowed values, i.e. "mysql" at this point?

Reply:
I can add that